dotcom-rendering/src/web/components/FilterKeyEventsToggle.importable.tsx

```diff
@@ -31,7 +39,7 @@ export const FilterKeyEventsToggle = ({ filterKeyEvents }: Props) => {
 
 	return (
 		<>
-			<span id="filter-toggle" />
+			<span id="filter-toggle" css={spanStyles} />
```

Comment:
Can we not add styles to this `span`? As I understand it, this element purely exists to enable us to scroll the reader to this position but if we're not adding css to this it creates a coupling we probably don't want.